compose/galaxy-init/Dockerfile

```diff
@@ -2,15 +2,16 @@
 #
 # VERSION       Galaxy-central
 
-FROM quay.io/bgruening/galaxy-base:dev
+ARG GALAXY_BASE_TAG=dev
+FROM quay.io/bgruening/galaxy-base:${GALAXY_BASE_TAG}
 
 MAINTAINER Björn A. Grüning, bjoern.gruening@gmail.com
 
-ARG GALAXY_RELEASE
-ARG GALAXY_REPO
+ARG GALAXY_RELEASE=dev
```

Comment:
Should that not be $GALAXY_BASE_TAG as well or why do we have both and not use just the GALAXY_RELEASE tag?

Reply:
Should, but it doesn't work. After the FROM line, the variable vanishes.
Try to build this Dockerfile: 
```
ARG BUILD_VAR=dev
FROM quay.io/bgruening/galaxy-base:${BUILD_VAR}

RUN echo BUILD_VAR=$BUILD_VAR
```